src/index.html

```diff
@@ -11,7 +11,151 @@
+            name="fist-name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-label">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth-date"
+            class="date-input"
+          >
+        </label>
+        <label class="form-label">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label>
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            multiple
+          >
+        </label>
+        <label class="form-label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <label>
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            id="no"
+          >
+          <label for="no">No</label>
+        </label>
+        <label class="form-label">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="form-label">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="bedtime"
+            class="time-input"
+            step="2"
+          >
+        </label>
+        <label class="form-label">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            multiple size="4"
+            class="cars-selector"
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-label">
+          How do you rate our work
+          <input
+            class="slider"
+            type="range"
+            name="work-rating"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Additional info:</legend>
+        <label>
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+          ></textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            cols="20"
            rows="2"
        ></textarea>
```
try to fix it in such way

<img width="558" alt="Screenshot 2022-11-18 at 10 24 09" src="https://user-images.githubusercontent.com/94289038/202655659-2c739525-45e6-48f3-816e-d1cd2267364d.png">

Reply:
Unfortunately, that didn`t help. Mistake is still the same
![image](https://user-images.githubusercontent.com/62021566/202662019-fc067724-630a-44f5-bd71-e7f194b66b41.png)